scripts/build/termux_step_get_dependencies.sh

```diff
@@ -65,6 +72,12 @@ termux_step_get_dependencies() {
 				continue
 			elif [ "$PKG" = "ERROR" ]; then
 				termux_error_exit "Obtaining buildorder failed"
+			elif [ "$PKG" = "ghc" ] || [ "$PKG" = "ghc-libs" ] || [ "$PKG" = "ghc-libs-static" ]; then
+				IS_GHC_REQUIRED=true
+			fi
+			# Store haskell packages to register them in the cross ghc package database later.
+			if [ "${PKG/haskell-/}" != "$PKG" ]; then
+				HASKELL_PKGS_TO_REGISTER+=("${PKG}")
 			fi
```

Comment:
Should be possible to create this type of array in a setup function as well, since TERMUX_PKG_DEPENDS should be available there as well